### ISSUE TITLE
support for ios/manual vpn config ios?

### ISSUE THREAD

**Issue by kocoman1** [tonarino/innernet#129]
support for ios/manual vpn config ios? does it exist? thx

**Comment by heyheyhello** [comment]
See #3

**Comment by mcginty** [comment]
Yeah, we will probably just want a guide/quick tool to export "vanilla WireGuard" config files for devices that don't have support for the innernet client.
